core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -114,14 +114,11 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setCategory = async function(category) {
-    await action.click('Neutral element', neutralElement);
-    await action.waitForAutosave();
     await waitFor.presenceOf(
       explorationCategoryInput, 'Category input takes too long to be visible.');
     await (
       await forms.AutocompleteDropdownEditor(explorationCategoryInput)
     ).setValue(category);
-    await action.click('Neutral element', neutralElement);
```

Comment:
Ditto.

Reply:
After the category was selected and the drop down was closed, I don't think there is any reason to open the drop  down again. 

BTW, if we were having this after the deletion of first neutral element, it was leading to a failure.